nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -340,6 +340,12 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `diamond` package has been update from 0.8.36 to 2.0.15. See the [upstream release notes](https://github.com/bbuchfink/diamond/releases) for more details.
 
+- The `aseprite` package has been upgraded from 1.2.16.3 to 1.2.40. It now only
+  includes the unfree version and the old `aseprite-unfree` version is now just
+  an alias of the same package. The free version has been dropped because it
+  is EOL, and a maintained fork of the last free version of Aseprite, named
+  'LibreSprite', is available in nixpkgs from the `libresprite` package.
```

Comment:
```suggestion
- The `aseprite-unfree` package has been upgraded from 1.2.16.3 to 1.2.40.
  The free version of aseprite has been dropped because it is EOL
  and the package attribute now points to the unfree version.
  A maintained fork of the last free version of Aseprite, named
  'LibreSprite', is available in the `libresprite` package.
```